finetune_user_model.py

```diff
@@ -98,7 +92,8 @@ def main(model, user_hash, config: configs.BaseConfig):
     }
 
     if sys.gettrace() is not None:
-        main(base_configuration)
+        raise NotImplementedError
     else:
+        raise NotImplementedError
         sweep_id = wandb.sweep(sweep=sweep_configuration, project="adaptive-hci-offline-adaptation")
         wandb.agent(sweep_id, function=main, count=10)
```

Comment:
how do we deal with when the file gets called? do we avoid that completely and don't do anything or make sure it runs as a separate thing?

Reply:
let's deal with it in another pull request

Reply:
I've put the not implemented error because i didn't know how that would work, i still need to fully understand how you do the sweeping